packaging/installer/methods/packages.md

```diff
@@ -27,26 +57,38 @@ Available groups:
 | `repoconfig` | Configuration packages        |
 | `devel`      | Dev builds (ignore)           |
 
-Supported distributions:
+### Supported Distributions
+
+Within each repository group, you'll find directories for specific distributions:
 
-- `amazonlinux`
-- `el` (RHEL, CentOS, AlmaLinux, Rocky Linux)
-- `fedora`
-- `ol` (Oracle Linux)
-- `opensuse`
+| Repository Directory | Primary Distribution | Compatible Distributions |
+|---------------------|----------------------|--------------------------|
+| `amazonlinux` | Amazon Linux | Compatible Amazon-based distros |
+| `el` | Red Hat Enterprise Linux | CentOS, AlmaLinux, Rocky Linux |
+| `fedora` | Fedora | Compatible Fedora-based distros |
+| `ol` | Oracle Linux | Compatible Oracle-based distros |
+| `opensuse` | openSUSE | Compatible SUSE-based distros |
```

Comment:
```suggestion
| `amazonlinux` | Amazon Linux | Binary-compatible Amazon Linux based distros |
| `el` | Red Hat Enterprise Linux | CentOS, AlmaLinux, Rocky Linux, and other binary-compatible distros |
| `fedora` | Fedora | Binary-compatible Fedora-based distros |
| `ol` | Oracle Linux | Binary-compatible Oracle Linux based distros |
| `opensuse` | openSUSE | Binary-compatible SUSE-based distros |
```

Specifying that it’s binary compatibility that matters is not as significant here as for the DEB-based distributions, but it’s still a good idea to be specific about this.